app/stats/Metrics.tsx

```diff
@@ -66,7 +69,7 @@ const ServiceSection = ({
   );
 };
 
-const MetricBox = ({
+export const MetricBox = ({
```

Comment:
Tady ještě můžem zase smáznout ten `export`?

Reply:
(Logika je taková, že když tam není export, máš jistotu, že to nepoužívá nikdo jiný a nemusíš se bát to měnit.)

Reply:
rozumiem, zmazane :) a suhlasim s merge